src/sdp/RemoteSdp.cpp

```diff
@@ -85,11 +85,15 @@ namespace mediasoupclient
 		if (iceParameters.find("iceLite") != iceParameters.end())
 			sdpObject["icelite"] = "ice-lite";
 
-		for (auto& kv : this->mediaSections)
+		size_t idx = 0;
+		for (auto* mediaSection : this->mediaSections)
```

Comment:
Doesn't look better to just use `for (auto idx{ 0u }; idx < this->mediaSection.size(); ++idx)`?